src/+FeatureCreators/LCDFeatureSet.m

```diff
@@ -0,0 +1,202 @@
+classdef LCDFeatureSet < FeatureCreators.Base
+% Largest Common Denominator feature set Specifies a feature set consisting of:
+%   see LCDFeatureSet.getAFErequests()
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        deltasLevels;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Access = public)
+        
+        function obj = LCDFeatureSet( )
+            obj = obj@FeatureCreators.Base();
+        end
+        %% ----------------------------------------------------------------
+
+        function afeRequests = getAFErequests( obj )
+            commonParams = {...
+                'pp_bNormalizeRMS', true, ...

```

Comment:
bNormalizeRMS won't have much an effect on localisation as localisation features are Gaussian normalised anyway. But you are right pp_bBinauralRMS will be an issue as it destroys ILD, right?

Sounds like at best we could converge to two sets of ratemaps/preprocessors? That's probably still worth doing.